pkg/features/kube_features.go

```diff
@@ -180,6 +187,7 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ExposeFlavorsInLocalQueue:           {Default: true, PreRelease: featuregate.Beta},
 	AdmissionCheckValidationRules:       {Default: false, PreRelease: featuregate.Deprecated},
 	KeepQuotaForProvReqRetry:            {Default: false, PreRelease: featuregate.Deprecated},
+	ManagedJobsNamespaceSelector:        {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Please also update [The currently supported features table](https://github.com/kubernetes-sigs/kueue/blob/dff92c9bcb5411d2b12bbf3240c977361d0ef87d/site/content/en/docs/installation/_index.md?plain=1#L234)

Reply:
Table updated.